Userland/Libraries/LibGfx/ClassicStylePainter.cpp

```diff
@@ -9,6 +9,7 @@
  */
 
 #include <AK/StringView.h>
+#include <LibGUI/TabWidget.h>
```

Comment:
The commit title/body isn't quite accurate, we're still including the full TabWidget.h in LibGfx. It's a bit unfortunate - maybe TabPosition belongs in LibGfx (but that can be a future FIXME, not asking you to do that here).